src/components/calculator.js

```diff
@@ -1,41 +1,80 @@
+      <div className="operators-buttons calc-button-style" role="button" tabIndex="0" onClick={() => { updateCalculation(calculatorObj, 'x', setCalculatorObj); }} onKeyPress={() => { updateCalculation(calculatorObj, 'x', setCalculatorObj); }}>x</div>
+      <div className="calc-button-style" role="button" tabIndex="0" onClick={() => { updateCalculation(calculatorObj, '4', setCalculatorObj); }} onKeyPress={() => { updateCalculation(calculatorObj, '4', setCalculatorObj); }}>4</div>
+      <div className="calc-button-style" role="button" tabIndex="0" onClick={() => { updateCalculation(calculatorObj, '5', setCalculatorObj); }} onKeyPress={() => { updateCalculation(calculatorObj, '5', setCalculatorObj); }}>5</div>
+      <div className="calc-button-style" role="button" tabIndex="0" onClick={() => { updateCalculation(calculatorObj, '6', setCalculatorObj); }} onKeyPress={() => { updateCalculation(calculatorObj, '6', setCalculatorObj); }}>6</div>
+      <div className="operators-buttons calc-button-style" role="button" tabIndex="0" onClick={() => { updateCalculation(calculatorObj, '-', setCalculatorObj); }} onKeyPress={() => { updateCalculation(calculatorObj, '-', setCalculatorObj); }}>-</div>
+      <div className="calc-button-style" role="button" tabIndex="0" onClick={() => { updateCalculation(calculatorObj, '1', setCalculatorObj); }} onKeyPress={() => { updateCalculation(calculatorObj, '1', setCalculatorObj); }}>1</div>
+      <div className="calc-button-style" role="button" tabIndex="0" onClick={() => { updateCalculation(calculatorObj, '2', setCalculatorObj); }} onKeyPress={() => { updateCalculation(calculatorObj, '2', setCalculatorObj); }}>2</div>
+      <div className="calc-button-style" role="button" tabIndex="0" onClick={() => { updateCalculation(calculatorObj, '3', setCalculatorObj); }} onKeyPress={() => { updateCalculation(calculatorObj, '3', setCalculatorObj); }}>3</div>
+      <div className="operators-buttons calc-button-style" role="button" tabIndex="0" onClick={() => { updateCalculation(calculatorObj, '+', setCalculatorObj); }} onKeyPress={() => { updateCalculation(calculatorObj, '+', setCalculatorObj); }}>+</div>
+      <div className="zero-button calc-button-style" role="button" tabIndex="0" onClick={() => { updateCalculation(calculatorObj, '0', setCalculatorObj); }} onKeyPress={() => { updateCalculation(calculatorObj, '0', setCalculatorObj); }}>0</div>
+      <div className="calc-button-style" role="button" tabIndex="0" onClick={() => { updateCalculation(calculatorObj, '.', setCalculatorObj); }} onKeyPress={() => { updateCalculation(calculatorObj, '.', setCalculatorObj); }}>.</div>
+      <div className="operators-buttons calc-button-style" role="button" tabIndex="0" onClick={() => { updateCalculation(calculatorObj, '=', setCalculatorObj); }} onKeyPress={() => { updateCalculation(calculatorObj, '=', setCalculatorObj); }}>=</div>
+    </div>
```

Comment:
To follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md). I would suggest that we reduce the number of duplicated codes to follow the DRY principle (Do not Repeat Yourself) 🤓